PREPARATION.md

```diff
@@ -57,7 +57,7 @@ There are a few common issues that pop up when deploying the SCIM Bridge.
 * You should **never** need to log into this Provision Manager account manually. Please refrain from doing so.
 * Do not attempt to perform a provisioning sync until the setup has been completed.
 * Once set up, your Identity Provider becomes the _authoritative source_ of information for your 1Password accounts. With Provisioning enabled, you _cannot edit user details manually_, and must do so through your Identity Provider.
-
+* The 1Password SCIM Bridge is not considered a service that requires high-availability. Unless you have a specific need to do so, you should only need a single instance of the service operating at a time.
```

Comment:
Reading this again, I would remove the `specific need to do so` part. Running multiple SCIM Bridges is explicitly unsupported and not a case we test against.